binding/Binding.Shared/LibraryLoader.cs

```diff
@@ -192,7 +191,7 @@ private static class Mac
 
 		private static class Linux
 		{
-			private const string SystemLibrary = "libdl.so";
+			private const string SystemLibrary = "dl";
```

Comment:
@mattleibow apparently this throws DllNotFoundException on a somewhat clean Ubuntu 20.04 installation